Mathlib/Algebra/Order/Hom/Ring.lean

```diff
@@ -416,6 +416,14 @@ theorem symm_trans_self (e : α ≃+*o β) : e.symm.trans e = OrderRingIso.refl
 
 end LE
 
+variable {R S : Type*} [Mul R] [Add R] [Mul S] [Add S] [Preorder R] [Preorder S]
```

Comment:
These variables are now globally in the file. Can you put all this change in a section, like the sections just above and just below this code?